src/physics/em/detail/SBEnergyDistHelper.hh

```diff
@@ -211,40 +200,22 @@ CELER_FUNCTION auto SBEnergyDistHelper::calc_max_xs(const SBTables& xs_params,
     const real_type x_frac = calc_xs_.x_fraction();
     MaxXs           result;
 
-    // Calc xs
-    {
-        auto get_value = [&xs_params, &el](size_type ix) -> real_type {
-            // Index of the largest xs for exiting energy for the given
-            // incident grid point
-            size_type iy = xs_params.sizes[el.argmax[ix]];
-            // Value of the maximum cross section
-            return xs_params.reals[el.grid.at(ix, iy)];
-        };
-
-        result.xs = (1 - x_frac) * get_value(x_idx)
-                    + x_frac * get_value(x_idx + 1);
-    }
-
-    // Calc energy
-    {
-        // The 'y' grid is fractional exiting energy
-        const NonuniformGrid<real_type> ee_grid{el.grid.y, xs_params.reals};
-
-        auto get_value = [&xs_params, &el, &ee_grid](size_type ix) -> real_type {
-            // Index of the largest xs for exiting energy for the given
-            // incident grid point
-            size_type iy = xs_params.sizes[el.argmax[ix]];
-            // Value of the exiting energy grid
-            return ee_grid[iy];
-        };
-
-        real_type efrac = (1 - x_frac) * get_value(x_idx)
-                          + x_frac * get_value(x_idx + 1);
-        result.energy = efrac * inc_energy;
-    }
+    // Calc max xs
+    auto get_value = [&xs_params, &el](size_type ix) -> real_type {
+        // Index of the largest xs for exiting energy for the given
+        // incident grid point
+        size_type iy = xs_params.sizes[el.argmax[ix]];
+        // Value of the maximum cross section
+        return xs_params.reals[el.grid.at(ix, iy)];
+    };
+    result.xs = (1 - x_frac) * get_value(x_idx) + x_frac * get_value(x_idx + 1);
+
+    // Calc max scaled xs
+    result.scaled_xs = (1 - x_frac) * xs_params.reals[el.grid.at(x_idx, 0)]
```

Comment:
Maybe rename to `xs_zero` (or `xs0` or whatever) to avoid confusion? No scaling has yet been applied, and this is the cross section for producing zero-energy gammas, regardless of whether scaling is being used.